### PR TITLE
Add types for `getOrder` and fix types for `ResultsResponse`

### PR DESCRIPTION
**Pull request by ItzCrazyKns**

### CODE REVIEW

**Review by swellmike — APPROVED** [review]

File: types/index.d.ts [hunk 1]
```diff
@@ -166,6 +166,7 @@ export namespace cart {
   function submitOrder(): Promise<Order>;
   function updateItem(id: string, input: CartItem): Promise<Cart>;
   function update(input: object): Promise<Cart>;
+  function getOrder(input?: string): Promise<Order>
```
Comment (swellmike) [inline types/index.d.ts:169]: I might call `input` => `checkoutId`.

Comment (ItzCrazyKns) [reply types/index.d.ts]: Yes, I have updated those and I have also fixed the types for `ResultsResponse`, If I see any other missing type I will definitely open a pull request.

File: types/index.d.ts [hunk 2]
```diff
@@ -88,11 +88,12 @@ export interface ResultsResponse<T> {
   count: number;
   page: number;
   pages?: {
-    [index: number]: {
-      start: number;
-      end: number;
-    };
+      [index: number]: {
```
Comment (swellmike) [inline types/index.d.ts:91]: can you preserve the original indentation?  The `prettier` config should limit it to 2 spaces

Comment (ItzCrazyKns) [reply types/index.d.ts:91]: yes of course

**Pull request merged**